jobs/e2e_node/OWNERS

```diff
@@ -4,3 +4,8 @@ approvers:
 - Random-Liu
 - yguo0905
 - yujuhong
+- tallclair
+- derekwaynecarr
+- ConnorDoyle
+- klueska
+- dchen1107
```

Comment:
Is this the list of OWNERS for the kubelet package or all sub-packages under it? It doesn't seem like either...

Reply:
@yujuhong I didnt remove any existing right, but just added the names from the /test/e2e_node/OWNERS listing here:
https://github.com/kubernetes/kubernetes/blob/master/test/e2e_node/OWNERS